app/templates/_Makefile

```diff
@@ -64,6 +66,9 @@ $(NODE_MODULES): $(PACKAGE_JSON) ## Install node_modules
 	@$(YARN)
 	@touch $(NODE_MODULES)
 
+.PHONY: document
+document: $(DOCUMENT) $(ALL_FILES)
```

Comment:
What about calling this just `docs`, or `apidocs`? Document sounds a little too generic